### ISSUE TITLE
Allow two different playback behaviour

### ISSUE THREAD

**Issue by toto-polo** [musescore/MuseScore#18215]
### Your idea

I'm a professional composer. I would like to be able, as in most DAWs, to have two playback behaviors:

- A continuous playback mode, which is currently the one chosen in Musescore;
- A return to last selection mode: when you restart playback, playback return at the point you previously selected, until you select another note from which playback will start.

Example: you select the first beat of bar 4. Until you select another note, playback will always return from the first beat of bar 4.
![playback_modes](https://github.com/musescore/MuseScore/assets/39052403/425465c1-da47-4122-8357-e7c4db3ea455)

### Problem to be solved

The actual playback engine has just on way to play the score. You can tinker to return to a certain position using loop mode, but this is not practical, as you have to keep selecting a loop start and end everytime. 

In my DAWs, I never use continuous playback.

**Comment by cbjeukendrup** [comment]
Alternative suggestion: an extra play command like "play from last start position". Some apps have many options, like "play from start", "play from selection", "play from playback cursor position", and "play from last start position".
Then, every user can assign the spacebar keyboard shortcut to their favourite play option, and maybe a different shortcut like Shift+Space to another command...

**Comment by toto-polo** [comment]
That could be an idea. But hit something like _alt + spacebar_ (or whatever shortcut) for an extra playback behaviour sounds unnatural if you use it all days. Instinctivly you just hit _spacebar_ to play on every sofware. If we choose your idea, maybe we have to let user choosing wich playback behaviour he wants for _spacebar_ and _alt + spacebar_

**Comment by cbjeukendrup** [comment]
Yes, all keyboard shortcuts can be customised anyway (Preferences > Keyboard shortcuts), so you can swap them if you want.

**Comment by iainhallam** [comment]
Great idea. While arranging I constantly want to play from the last start position, so being able to reassign the spacebar to this would be perfect. Using a different key for the default behaviour would be useful too.

**Comment by iainhallam** [comment]
I realised today that the original request is asking for two actions that can be taken in many applications - pause and stop. There are currently shortcuts for Play (Space) and Pause (also Space), but no predefined shortcut for the Stop action that is already in the list, so I assigned it to a keyboard shortcut and used it while playing. It seems to have exactly the same effect as the Pause action; is it designed to do something different? Seems like making Stop reset the playback position to the previous location would sort out the original request.

This still leaves the issue that clicking on any item after using Stop would reset the playback position to whatever was just clicked on, so we'd still need an action to Play From Last Start Position in case someone edited a note then wanted to hear the change without having to continually update the loop markers.

In fact, do we need to consider adding a playback cursor that the user can move independently of the editing cursor, as I know Sibelius does?

**Comment by scorster** [comment]
There's a similar request here:

https://musescore.org/en/node/359132

**Comment by iainhallam** [comment]
#17376 has less discussion, but is assigned and in state "needs design".

**Comment by Lycalp** [comment]
Hello! 

Is there any progress about the feature request of having "play from the previous start position"? I REALLY hope, this feature will be available very soon in Musescore Studio...
